web/models/audit_log.ex

```diff
@@ -0,0 +1,31 @@
+defmodule HexWeb.AuditLog do
+  use HexWeb.Web, :model
+
+  schema "audit_logs" do
+    belongs_to :actor, User
+    field :action, :string
+    field :extra_params, :map
```

Comment:
Sounds good, fixed in: d73f282

2016-02-28 5:06 GMT+01:00 Eric Meadows-Jönsson <notifications@github.com>:

> In web/models/audit_log.ex
> <https://github.com/hexpm/hex_web/pull/238#discussion_r54343921>:
>
> > @@ -0,0 +1,31 @@
> > +defmodule HexWeb.AuditLog do
> > +  use HexWeb.Web, :model
> > +
> > +  schema "audit_logs" do
> > +    belongs_to :actor, User
> > +    field :action, :string
> > +    field :extra_params, :map
>
> Agree that it can simply be params.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hexpm/hex_web/pull/238/files#r54343921>.
>



-- 
Wojtek Mach